scopes/harmony/ui/aspect-box/aspect-box.tsx

```diff
@@ -11,12 +16,10 @@ export type AspectBoxProps = {
   data: any;
 } & React.HTMLAttributes<HTMLDivElement>;
 
+const collapsedIcon = 'https://static.bit.dev/bit-icons/collapse.svg';
+const expandIcon = 'https://static.bit.dev/bit-icons/expand.svg';
```

Comment:
we're doing the same in component drawer so maybe we can make a component and use in both?